test/setup/outputs.tf

```diff
@@ -58,3 +58,7 @@ output "group_name" {
 output "service_account_email" {
   value = google_service_account.int_test.email
 }
+
+output "policy_id" {
```

Comment:
If we use the gcloud command to evaluate the policy id during the prepare step, we could probably get rid of this.